Core/src/Upload/ResumableUploader.php

```diff
@@ -264,7 +264,7 @@ protected function getStatusResponse()
         $request = new Request(
             'PUT',
             $this->resumeUri,
-            ['Content-Range' => 'bytes */*']
+            ['Content-Range' => 'bytes */' . $this->data->getSize()]
```

Comment:
Isn't this change in #7100 as well?

Reply:
Yes, first #7100 should be merged then this change will go away.